custom/icds_reports/views.py

```diff
@@ -2049,6 +2058,11 @@ def post(self, request, *args, **kwargs):
         year = int(request.POST.get('year', None))
         selected_date = date(year, month, 1).strftime('%Y-%m-%d')
 
+        if state_id and not user_can_access_location_id(
```

Comment:
Why doesn't this also check for full org permissions if state_id isn't present?

Reply:
These only exist at the state level, you can't download nationally so leaving that out will cause the check at https://github.com/dimagi/commcare-hq/pull/26641/files/b7ac5540601cdf914de7058f64f851957763e2d0#diff-129264561592e648289c087ab1350b9bR2067 to fail